<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bosswindow</class>
 <widget class="QMainWindow" name="bosswindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>蛋壳行记</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>700</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::East</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>工作手册</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-35</y>
        <width>760</width>
        <height>731</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/image/images/timg.jpg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bLogout">
      <property name="geometry">
       <rect>
        <x>815</x>
        <y>190</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>退出登录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bExit">
      <property name="geometry">
       <rect>
        <x>815</x>
        <y>330</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>退出系统</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>560</y>
        <width>180</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt; font-weight:600;&quot;&gt;Page 1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>我家客房</string>
     </attribute>
     <widget class="QPushButton" name="broomQuery1">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>20</y>
        <width>80</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
     <widget class="QPushButton" name="broomShow">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>20</y>
        <width>130</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>显示全部</string>
      </property>
     </widget>
     <widget class="QPushButton" name="broomEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>580</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>编辑客房</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lroomQuery">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>60</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;名称&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="rTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>940</width>
        <height>470</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>20</y>
        <width>60</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;类型&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cQuery">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>20</y>
        <width>170</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;等线&quot;;</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>选择房间类型</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>单人间</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>大床房</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>标间</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>家庭房</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>套房</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="broomQuery2">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>20</y>
        <width>80</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
     <widget class="QPushButton" name="broomAdd">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>580</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>添加客房</string>
      </property>
     </widget>
     <widget class="QPushButton" name="broomDel">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>580</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>删除客房</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>580</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; font-weight:600;&quot;&gt;Page 2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Tab1">
     <attribute name="title">
      <string>我家酒店</string>
     </attribute>
     <widget class="QPushButton" name="bhotelQuery1">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>20</y>
        <width>80</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bhotelQuery2">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>20</y>
        <width>70</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bhotelShow">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>20</y>
        <width>150</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>显示全部</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bhotelEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>580</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>编辑酒店</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bhotelAdd">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>580</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>创建酒店</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bhotelDel">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>580</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>删除酒店</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>60</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;名称&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lhotelQuery">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="hTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>940</width>
        <height>470</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cQuery2">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>20</y>
        <width>120</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;等线&quot;;</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>选择地区</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>海淀区</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>朝阳区</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>丰台区</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>东城区</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>西城区</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>50</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;位置&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>570</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; font-weight:600;&quot;&gt;Page 3&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>我家经理</string>
     </attribute>
     <widget class="QPushButton" name="bmanAdd">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>580</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>添加经理</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bmanQuery">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>20</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bmanShow">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>20</y>
        <width>150</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>显示全部</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bmanDel">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>580</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>删除经理</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;经理账号&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lmanQuery">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="manTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>940</width>
        <height>470</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>580</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; font-weight:600;&quot;&gt;Page 4&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>我家顾客</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>111</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;顾客账号&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lcusQuery">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="bcusQuery">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>20</y>
        <width>80</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bcusShow">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>20</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>全部显示</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bcusDel">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>580</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>删除顾客</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="cusTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>950</width>
        <height>470</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>580</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; font-weight:600;&quot;&gt;Page 5&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
